docs/Contributors.md

```diff
@@ -1,7 +1,7 @@
 # Passr Contributors List
 
 Braiden 
-Dina
+Dina [polygonart6777](https://github.com/polygonart6777)
 Isaiah
 Noah
 Samantha
```

Comment:
I want to add a past contributors section to recognize people that commited work at one time

Reply:
Done!